scripts/task_test_jit_cache_package_build_import.sh

```
@@ -72,22 +368,135 @@ python -c "import torch; print(torch.__version__)"
+
 echo ""
 echo "========================================"
 echo "Building flashinfer-jit-cache wheel"
 echo "========================================"
 cd flashinfer-jit-cache
-python -m build --wheel
+run_with_aot_memory_monitor "build_flashinfer_jit_cache_wheel" \
+    python -m build --wheel
 
 # Get the built wheel file
 WHEEL_FILE=$(ls dist/*.whl | head -n 1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)